ksa_compliance/ksa_compliance/doctype/sales_invoice_additional_fields/sales_invoice_additional_fields.py

```diff
@@ -340,17 +340,14 @@ def add_integration_log_document(self, zatca_message, integration_status, zatca_
     @property
     def qr_image(self):
         if self.qr_code:
-            qr = qrcode.QRCode(version=1, box_size=10, border=5)
-            qr.add_data(self.qr_code)
-            qr.make(fit=True)
-            img = qr.make_image(fill_color="black", back_color="white")
-            img_buffer = BytesIO()
-            img.save(img_buffer, format='PNG')
-            img_buffer.seek(0)
-            img_str = base64.b64encode(img_buffer.read()).decode("utf-8")
+            qr = pyqrcode.create(self.qr_code)
+            with BytesIO() as buffer:
+                qr.png(buffer, scale=7)
+                buffer.seek(0)
+                img_str = base64.b64encode(buffer.getvalue()).decode("utf-8")
             return img_str
         else:
-            return "No QR image"
+            return None
```

Comment:
We can now invert the if and exit early (i.e. `if not self.qr_code:` and exit the function to unindent the rest)